src/screen/itemDetail/index.tsx

```diff
@@ -1,21 +1,22 @@
 import React from "react";
 import{View,Text,ImageBackground,SafeAreaView,FlatList} from 'react-native'
-import { connect } from "react-redux";
+import {useDispatch} from "react-redux";
 import { addToCart,addToWishlist } from "redux/shopping/action"; 
 import styles from "./styles";
 import Icon from 'react-native-vector-icons/FontAwesome'
 import {Button} from "components";
-import {colors, size} from "theme"
+import {Colors, size} from "theme"
 
-const ItemDetail = ({navigation,route, addToCart, addToWishlist}) =>{
+const ItemDetail = ({navigation,route}) =>{
     const item = route.params
     const sizeCheck = item.size
```

Comment:
Can you find an alternative / better approach to use sizeCheck in the FlatList ?